common/plugins/eu.esdihumboldt.hale.common.headless/src/eu/esdihumboldt/hale/common/headless/transform/filter/InstanceFilterDefinition.java

```diff
@@ -159,10 +163,7 @@ private boolean applyFilters(Instance instance) {
 			return false;
 
```

Comment:
I think there is still a case not handled: If there is only excluded types, no other filters, all instances will be rejected.